secrets/Makefile

```diff
@@ -11,8 +11,7 @@ $(info CONCENT_SECRET_DIR=$(CONCENT_SECRET_DIR))
 
 
 CLUSTER_SECRETS :=                                            \
-    $(CONCENT_SECRET_DIR)/secrets.py                          \
```

Comment:
`secrets.py` should have been removed in the branch that creates a separate deployment for secrets.

Reply:
This branch add a `secrets.py.j2` template for generating `secrets.py`. This is the right place for this change.